libs/community/langchain_community/llms/ipex_llm.py

```diff
@@ -1,5 +1,5 @@
 import logging
```

Comment:
Do not forget to fix the style errors :)

Reply:
Fixed.